WORKSPACE

```diff
@@ -14,9 +14,9 @@ new_local_repository(
 
 http_archive(
     name = "six",
-    build_file = "@//:six.BUILD",
-    sha256 = "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a",
-    urls = ["https://pypi.python.org/packages/source/s/six/six-1.10.0.tar.gz#md5=34eed507548117b2ab523ab14b2f8b55"],
+    build_file = "@//:third_party/six.BUILD",
+    sha256 = "d16a0141ec1a18405cd4ce8b4613101da75da0e9a7aec5bdd4fa804d0e0eba73",
+    urls = ["https://pypi.python.org/packages/source/s/six/six-1.12.0.tar.gz"],
 )
```

Comment:
I don't think this `http_archive` is required in `WORKSPACE`, it's already loaded by `protobuf_deps()` a few lines above.

Reply:
I've just left the existing declaration locations as-is, but it'd be worth considering removing the duplicate.

Reply:
Added commit